proto/api/v1/operations.proto

```diff
@@ -52,6 +58,10 @@ message ListOperationsResponse {
   repeated ListOperationItem active_operations = 2;
   // List of the scheduler finished (with error or not) operations.
   repeated ListOperationItem finished_operations = 3;
```

Comment:
Hey @mflilian what do you think to return only operations without saying which state they are in?